types/theme.d.ts

```diff
@@ -20,7 +21,15 @@ type ObjectOf<T> = { [key: string]: T }
 // Props
 // ========================================================
 
-export type IProps = ObjectOf<any>
+export type IProps = Extendable<{
+  variables?: ComponentVariablesInput
```

Comment:
this won't work for all the components, unfortunately (`Provider`) - lets keep to be generic for now

Reply:
ok, done